mula/scheduler/schedulers/normalizer.py

```diff
@@ -221,22 +221,21 @@ def push_task(self, normalizer: Plugin, raw_data: RawData, caller: str = "") ->
         score = self.ranker.rank(
             SimpleNamespace(
                 raw_data=raw_data,
-                task=task,
+                task=normalizer_task,
             ),
         )
 
-        # We need to create a PrioritizedItem for this task, to
-        # push it to the priority queue.
-        p_item = PrioritizedItem(
-            id=task.id,
+        # TODO: check the correct attributes, schema
+        task = Task(
             scheduler_id=self.scheduler_id,
             priority=score,
-            data=task.model_dump(),
-            hash=task.hash,
+            hash=normalizer_task.hash,
+            data=normalizer_task.model_dump(),
```

Comment:
Perhaps it makes sense to rename either `data` or `normalizer_task` so that it makes sense that `task.data == normalizer_task.model_dump()`? Either `task.arguments` and `normalizer_arguments` or `task.meta` and `normalizer_meta` or just `normalizer_task_data`?

Reply:
Yeah will do. Should've prefaced that the normalizer scheduler needed some work.